picard/ui/metadatabox/__init__.py

```diff
@@ -282,19 +285,61 @@ def _copy_value(self):
                 except (TypeError, ValueError) as why:
                     log.warning(why)
                     value = ['']
-
-            if value is not None:
-                self.tagger.clipboard().setText(MULTI_VALUED_JOINER.join(value))
-                self.clipboard = value
+            return value
+
+        #TODO: Support  copying data from multiple files
+
+        items = self.selectedItems()
+        if len(items) > 1:
+            # We have multiple tags selected, so copy them as a JSON string
+            data = {}
+            for item in items:
+                key = self.tag_diff.tag_names[item.row()]
+                if key not in data:
+                    data[key] = {}
+
+                fieldnames = {self.COLUMN_TAG: 'tag_name', self.COLUMN_ORIG: 'original_value', self.COLUMN_NEW: 'new_value'}
+                fieldname = fieldnames[item.column()]
+
+                value = get_value_as_string(item)
+                if value and fieldname:
+                    data[key] = { fieldname: value } | data[key]
+            mimedata = QtCore.QMimeData()
+            mimedata.setData("application/json;contents=picard-tags", json.dumps(data).encode('utf-8'))
```

Comment:
Let's put `"application/json;contents=picard-tags"` into a constant and use it here and below in the pasting code.